lib/android-tools-info.ts

```diff
@@ -343,45 +337,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		return (() => {
 			if (!this.installedTargetsCache) {
 				try {
```

Comment:
I think we don't need this `try/catch`. You can move the logger.trace in the `if` instead of `throw new Error`and do the rest in the `else`.